### ISSUE TITLE
FE build process

### ISSUE THREAD

**Issue by RebeccaGreiner** [amosproj/amos2024ss05-knowledge-graph-extractor#22]
## User story
1. As a dev team
2. we need to streamline the build process for our project's front-end components,
3. So that we can efficiently compile, bundle, and optimize the codebase for deployment.

## Acceptance criteria
* Implement build system
* Demo of build process (Homework 3)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
